src/contracts/HiddenMintTest.sol

```diff
@@ -0,0 +1,40 @@
+pragma solidity ^0.8.0;
+
+ function mint(address account, uint256 amount) public {
+
+ _mint(account, amount)
+
+  }
+
+
+
+Function _mint(address account, uint256 amount) internal virtual OnlyOwner{
+
+   require(account != address(0), "ERC20: mint to the zero address");
+
+
+
+    _beforeTokenTransfer(address(0), account, amount);
+
+
+
+    _totalSupply = _totalSupply.add(amount);
+
+    _balances[account] = _balances[account].add(amount);
+
+    emit Transfer(address(0), account, amount);
+
+}
```

Comment:
In solidity functions can't be declared outside of a contract. It looks like you copy pasted this from another contract because functions like `_beforeTokenTransfer` and modifiers like `OnlyOwner` aren't defined. I would just use the contract you copied these functions from as the test case because that contract has the mint function.